faction/models.py

```diff
@@ -655,20 +656,28 @@ def updateLog(self):
         old = now - self.getHist("armory")
         news = self.news_set.all()
         news.filter(timestamp__lt=old).delete()
+        last_armory = 0
+        last_fund = 0
+        bulk_mgr = BulkCreateManager(chunk_size=20)
```

Comment:
This is the bit that needs testing in prod, I suspect 100 is going to be a better value here but went for 20 to be safe